info.json

```
@@ -1,14 +1,14 @@
 {
 	"name":"CircuitHUD-V2",
-	"version": "1.3.0",
+	"version": "2.0.0",
 	"title":"Circuit HUD V2",
 	"author":"Xiomax & feupeu",
```

```suggestion
	"author":"Xiomax & feupeu",
```
Please add your name here as you also deserve it!